sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/implementation/RxDocumentClientImpl.java

```diff
@@ -1716,12 +1722,13 @@ private Mono<RxDocumentServiceResponse> replace(RxDocumentServiceRequest request
     }
 
     private Mono<RxDocumentServiceResponse> patch(RxDocumentServiceRequest request, DocumentClientRetryPolicy documentClientRetryPolicy) {
-        populateHeaders(request, RequestVerb.PATCH);
```

Comment:
/facepalm

Reply:
I know, when we saw this, it was a facepalm reaction from all of us :D